bamoe-ddl/data-index.sql

```diff
@@ -0,0 +1,742 @@
+--

```

Comment:
@josedee I'd remove this comments

Reply:
You mean the pg_dump export comments or all comments